mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -269,6 +269,7 @@ def __send_sync(timeout=None):
                             event_queue.put((key, value, time()))
                             idx = sync_uuid_list.index(value)
                             logger.prn_inf("found SYNC in stream: {{%s;%s}} it is #%d sent, queued..."% (key, value, idx))
+                            connector.write_kv("__host_ack",sync_uuid)
```

Comment:
Pretty sure you mean `value` instead of `sync_uuid` here.
```suggestion
                            connector.write_kv("__host_ack", value)
```